### ISSUE TITLE
Disable a check

### ISSUE THREAD

**Issue by kennygrubb** [nickcolley/jest-axe#112]
I would like to disable the fallback role check. 

Using axe directly, I can do this.
```
axe.configure({
  checks: [
    {
      id: 'fallbackrole'
      enabled: false
    }
  ]
});
```

However, I don't see a way to configure checks through jest-axe. Thanks

**Comment by nickcolley** [comment]
Hello :)

You'd do this with the global configuration option: https://github.com/nickcolley/jest-axe#setting-global-configuration

```javascript
const axe = configureAxe({
  checks: [
    {
      id: 'fallbackrole'
      enabled: false
    }
  ]
})
```

Please re-open if this doesnt work for you :)

**Comment by dwightjack** [comment]
Hello, 

First of all, thanks for this great library.

I stumbled upon this issue while trying to add a custom rule to axe-core.

Looking to the source code, it seems to me that it is not possible to customize checks and add rules via the `configureAxe` because it supports options for the [axe.run](https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#options-parameter) method instead of the ones for `axe.configure`. 

Am I getting it wrong?

**Comment by nickcolley** [comment]
Hmm I think you're right, I'm not familiar with the difference between a 'check' and a 'rule'. If you can give me some examples of both and how they related I may be able to figure this out with you.

I'll re-open this. I also have limited time so would welcome a pull request if you figure it out without me.

**Comment by dwightjack** [comment]
I'm pretty new to axe custom rules as well. From what I understand, with a rule you match the elements that need to be tested, and with a check you actually check those elements. So to add a new rule you usually have to define both.

I might have some time next week to submit a PR with possible solutions. In the meanwhile importing `axe-core` and using `axe.configuration` before running tests seems to work, but that's prone to unexpected side effects I guess.

**Comment by nickcolley** [comment]
Thank you @kennygrubb for raising and @dwightjack for sorting, this is released as 3.5.0
